pkg/destroy/aws/aws.go

```diff
@@ -1597,18 +1601,40 @@ func deleteS3(session *session.Session, arn arn.ARN, logger logrus.FieldLogger)
 		Bucket: aws.String(arn.Resource),
 	})
 	err := s3manager.NewBatchDeleteWithClient(client).Delete(aws.BackgroundContext(), iter)
-	if err != nil {
+	if err != nil && !isBucketNotFound(err) {
```

Comment:
why is somethings like https://github.com/openshift/installer/blob/97ab75dbe6feb64f21063fb4f6a6e3996b5da6ed/pkg/destroy/aws/aws.go#L585-L589 not sufficeint.

Reply:
because in the case of the batch delete you're going to get something like:
awserr.Error -> (OrigError) s3manager.Errors[] -> s3.manager.Error (OrigError) -> awserr.Error

Reply:
+1